lib/matplotlib/figure.py

```diff
@@ -884,20 +884,20 @@ def add_axes(self, *args, **kwargs):
         rect : sequence of float
             A 4-length sequence of [left, bottom, width, height] quantities.
 
-        projection :
-            ['aitoff' | 'hammer' | 'lambert' | 'mollweide' | \
-'polar' | 'rectilinear'], optional
+        projection : {'aitoff', 'hammer', 'lambert', 'mollweide', \
+        'polar', 'rectilinear'}, optional
             The projection type of the axes.
 
         polar : boolean, optional
             If True, equivalent to projection='polar'.
 
-        This method also takes the keyword arguments for
-        :class:`~matplotlib.axes.Axes`.
+        **kwargs
+            This method also takes the keyword arguments for
+            :class:`~.axes.Axes`.
 
         Returns
-        ------
-        axes : Axes
+        -------
+        axes : :class:`~.axes.Axes`
```

Comment:
Types don't need to be marked up (just as you'd write `foo : int`).  Please leave as it was.

Reply:
Sometimes sphinx doesn't automatically put in links with out the markup though.

Reply:
Suite, I can always strip out unnecessary markup later...

Reply:
Reverted.